orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/job/PreconfiguredJobStage.groovy

```diff
@@ -72,17 +72,20 @@ class PreconfiguredJobStage extends RunJobStage {
         context[it] = preconfiguredMap[it]
       }
     }
```

Comment:
the code structure was a bit uncomfortable, so I restructured it a bit

Reply:
But the main idea is basically confirming that `mapping` is defined on the parameter before we use it